packages/contentstack-import/src/config/index.ts

```diff
@@ -388,7 +388,8 @@ const config: DefaultConfig = {
   developerHubBaseUrl: '',
   marketplaceAppEncryptionKey: 'nF2ejRQcTv',
   getEncryptionKeyMaxRetry: 3,
-  useNewModuleStructure: false,
+  useNewModuleStructure: true,
+  createBackupDir: './temp'
```

Comment:
Do we need to keep the default value as "temp"? I assume we'll only accept this key from an external configuration, right?

Reply:
@antonyagustine Yeah, I missed that. Added it for testing. Removed